### ISSUE TITLE
Support reading metadata with 4 CDJs on the network

### ISSUE THREAD

**Issue by EvanPurkhiser** [EvanPurkhiser/prolink-go#6]
Opening this to track research here, likely research into kuvo and tour devices.

**Comment by brunchboy** [comment]
I look forward to anything you can discover! But the literature of the Tour devices say that they lose this ability when there are any original nexus generation players on the network, so it is not likely we can do any better in terms of backwards compatibility.

**Comment by brunchboy** [comment]
Indeed! 😄 And after three days of exhausting searching and frustrating false starts leading to painful dead ends, I think I have finally found a great solution for building a CDJ-compatible NFS client in Java. You probably have a very different universe of options in Go, although it’s unfortunate that Google chose to roll their own protocol buffers rather than sticking with XDR. Everyone always loves to come up with a new data serialization format.
